geometrical_calculator_kuba.py

```diff
@@ -0,0 +1,393 @@
+import os
+import csv
+import math
+
+geometrical_calculator_history = []
+
+class Circle:    
+    
+    def __init__(self , r):
+        self.r = r
+                
+    def calculate_area(self):
+        return math.pi * self.r ** 2
+    
+    def calculate_circumference(self):
+        return 2 * math.pi * self.r
+    
+class Triangle:
+    
+    def __init__(self, a , b , c , h):
+        self.a = a
+        self.b = b
+        self.c = c
+        self.h = h
+        
+    def calculate_area(self):
+        return 1/2 * self.b * self.h
+    
+    def calculate_circumference(self):
+        return self.a + self.b + self.c
+      
+class Square:
+    
+    def __init__(self , a):
+        self.a = a
+        
+    def calculate_area(self):
+        return self.a ** 2
+    
+    def calculate_circumference(self):
+        return 4 * self.a
+        
+class Rectangle:
+    
+    def __init__(self , a , b):
+        self.a = a
+        self.b = b
+    
+    def calculate_area(self):
+        return self.a * self.b
+    
+    def caluclate_circumference(self):
+        return (2 * self.a) + (2 * self.b)
+    
+class Rhombus:
+    
+    def __init__(self , a , h):
+        self.a = a
+        self.h = h
+    
+    def calculate_area(self):
+        return self.a * self.h
+    
+    def calculate_circumference(self):
+        return 4 * self.a
+    
+class RegularPentagon:
+    
+    def __init__(self , a):
+        self.a = a
+    
+    def calculate_area(self):
+        return 0.25 * math.sqrt(5 * (5 + 2 * math.sqrt(5))) * self.a ** 2
+    
+    def calculate_circumference(self):
+        return 5 * self.a
+    
+class RegularHexagon:
+    
+    def __init__(self , a):
+        self.a = a
+    
+    def calculate_area(self):
+        return (3 * math.sqrt(3) / 2) * self.a ** 2
+    
+    def calculate_circumference(self):
+        return 6 * self.a
+    
+class RectangularPrism:
+    
+    def __init__(self , a , b , c):
+        self.a = a
+        self.b = b
+        self.c = c
+    
+    def calcuclate_area(self):
+        return 2 * (self.a * self.b) + 2 * (self.a * self.c) + 2 * (self.b * self.c)
+    
+    def calculate_circumference(self):
+        return (4 * self.a) + (4 * self.b) + (4 * self.c)
+    
+    
+def circle():
```

Comment:
Nie. Do praktycznie 300 linii chcemy to zmodyfikować. Jeżeli rozbijamy to na metody, a przy takiej kupie kodu wydaje mi się, że to spoko rozwiązanie, to zachowujemy podstawowe maniery i 1 metoda == 1 operacja, bo tak to alfy i omegi robią def geometrical_calculator(), który ma 300 linii kodu, 30 ifów i rozczytaj się z tym. 
Nawet jeżeli zrobimy to kosztem ilości kodu, bo nie wiem co miałem w głowie mówiąc, że to będzie 100 linii, to i tak chcemy w to iść, bo ciężko się w tym rozczytać.

Reply:
nie rozumiem jak to poprawic i gdzie